src/create-default-behavior.js

```diff
@@ -24,13 +24,11 @@ export default function createDefaultBehavior (editable) {
 
   return {
     focus (element) {
-      // Add a <br> element if the editable is empty to force it to have height
-      // E.g. Firefox does not render empty block elements and most browsers do
-      // not render  empty inline elements.
+      // Add an empty text node if the editable is empty to force it to have height
+      // E.g. Firefox does not render empty block elements
+      //   and most browsers do not render empty inline elements.
       if (!parser.isVoid(element)) return
-      const br = document.createElement('br')
-      br.setAttribute('data-editable', 'remove')
-      element.appendChild(br)
+      element.appendChild(document.createTextNode('\u0000'))
```

Comment:
I didn’t know! I’ve seen this happening before that an empty text node was folded away and best I could come up with was a `&nbsp;` but that too sometimes failed to work.

Reply:
Using `\u200B` would result in the same behavior, but we're using that zero width space already for something else: https://github.com/livingdocsIO/editable.js/blame/master/src/content.js#L87